Explorer/Packages/manifest.json

```diff
@@ -19,18 +19,18 @@
     "com.unity.cloud.ktx": "3.4.5",
     "com.unity.code-analysis": "0.1.2-preview",
     "com.unity.collections": "2.5.1",
-    "com.unity.ide.rider": "3.0.34",
+    "com.unity.ide.rider": "3.0.36",
     "com.unity.inputsystem": "1.13.1",
-    "com.unity.localization": "1.5.2",
+    "com.unity.localization": "1.5.4",
     "com.unity.mathematics": "1.3.2",
-    "com.unity.memoryprofiler": "1.1.5",
+    "com.unity.memoryprofiler": "1.1.6",
     "com.unity.multiplayer.center": "1.0.0",
     "com.unity.performance.profile-analyzer": "1.2.3",
-    "com.unity.project-auditor": "1.0.0-pre.1",
+    "com.unity.project-auditor": "1.0.1",
     "com.unity.render-pipelines.universal": "17.0.4",
     "com.unity.test-framework": "1.4.6",
     "com.unity.test-framework.performance": "3.0.3",
-    "com.unity.timeline": "1.8.7",
+    "com.unity.timeline": "1.8.8",
```

Comment:
Out of curiosity, is timeline even used?

Reply:
I thought about that too. I decided to do this PR first, and then do separate PRs for removing packages.